src/mito2/src/compaction.rs

```diff
@@ -362,9 +362,13 @@ impl CompactionScheduler {
 
         // Submit the compaction task.
         self.scheduler
-            .schedule(Box::pin(async move {
-                local_compaction_task.run().await;
-            }))
+            .schedule(Job::new(
+                "compaction",
+                Priority::Low(Some(Duration::from_secs(5 * 60))),
```

Comment:
Do we need to define a constant for this?

Reply:
> Do we need to define a constant for this?

fixed in f1354bf049